server/modlog.ts

```diff
@@ -171,7 +171,9 @@ export class Modlog {
 	async rename(oldID: ModlogID, newID: ModlogID) {
 		const streamExists = this.streams.has(oldID);
 		if (streamExists) await this.destroy(oldID);
-		await FS(`${this.logPath}/modlog_${oldID}.txt`).rename(`${this.logPath}/modlog_${newID}.txt`);
+		if (!this.getSharedID(oldID)) {
```

Comment:
is this change related? should it be in a separate PR?

Reply:
No, it's necessary; otherwise, trying to rename the battle crashes PS!